pyccel/codegen/printing/ccode.py

```diff
@@ -1105,9 +1105,9 @@ def _cast_to(self, expr, dtype, precision):
     def _print_DottedVariable(self, expr):
         """convert dotted Variable to their C equivalent"""
         if self.stored_in_c_pointer(expr.lhs):
-            code = '{}->{}'.format(self._print(ObjectAddress(expr.lhs)), self._print(expr.name))
+            code = f'{self._print(ObjectAddress(expr.lhs))}->{self._print(expr.name)}'
         else:
-            code =  '{}.{}'.format(self._print(expr.lhs), self._print(expr.name))
+            code =  f'{self._print(expr.lhs)}.{self._print(expr.name)}'
```

Comment:
This seems less readable to me like this. Please use intermediate variables to make it clearer. E.g:
```python
lhs_code = self._print(expr.lhs)
name_code = self._print(expr.name)
code = f'{lhs_code}.{name_code}'
```

Reply:
Both lines please